.circleci/config.yml

```diff
@@ -227,8 +227,7 @@ jobs:
                 command: |
                   source scripts/utils.sh
                   tag=v$(extract_version_from_current_wd)
-                  # Newlines (<br/>) are necessary for better layout on Marketplace, but make the formatting worse on GitHub
```

Comment:
Hmm now the change notes on Marketplace won't include `<br/>`s 🤔

Reply:
TBD if this comment still holds (you can try editing the existing change notes manually via Marketplace web, lmk if you need extra permissions for that)